### ISSUE TITLE
Cannot display OpenAPI Examples

### ISSUE THREAD

**Issue by dkirrane** [mrin9/RapiDoc#68]
I have some examples in my OpenAPI I want to use with `Try` functionality. 
But see `example-1` below does not display.

```
openapi: 3.0.2
info:
  version: '1.0'
  title: My OpenAPI
  description: OpenAPI with examples
servers:
  - url: /api
paths:
  /dogs:
    post:
      tags:
        - dog
      summary: Create a Dog
      operationId: createDog
      requestBody:
        description: A Dog
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/Dog'
            examples:
              example-1:
                value:
                  name: Fido
              example-2:
                value:
                  name: Dido
      responses:
        '201':
          description: Created
        '400':
          description: Bad Request
        '401':
          description: Unauthorized
        '403':
          description: Forbidden
      deprecated: false
components:
  schemas:
    Dog:
      type: object
      properties:
        name:
          type: string
          pattern: '^[A-Za-z]+'
      required:
        - name
```

**Comment by mrin9** [comment]
Thanks for reporting, will address this issue in our next patch.
> just for my note. It works for response but not for request-body

```
openapi: 3.0.2
info:
  version: '1.0'
  title: My OpenAPI
  description: OpenAPI with examples
servers:
  - url: /api
paths:
  /dogs:
    post:
      tags:
        - dog
      summary: Create a Dog
      requestBody:
        description: A Dog
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/Dog'
            examples:
              example-1:
                value:
                  name: Fido
              example-2:
                value:
                  name: Dido
      responses:
        '201':
          description: Created
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Dog'
              examples:
                example-1:
                  value:
                    name: Fido
                example-2:
                  value:
                    name: Dido
components:
  schemas:
    Dog:
      type: object
      properties:
        name:
          type: string
          pattern: '^[A-Za-z]+'
      required:
        - name
```

**Comment by mrin9** [comment]
had to release a urgent fix, this should have been taken care in `6.0.2`, let me know
